<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="customerstab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>751</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="customerstab_2">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButtoncustomers">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEditcustomer">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>write ID, name, adress or phonenumber</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetcustomers">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>711</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonsnewcustomers">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Create new</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonrefresh">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="itemstab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdititems">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>write ID, title, author, name or date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonsitems">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>711</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonsitems_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Create new</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonrefresh_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonsitems_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add item to cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonsitems_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Close app</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
